### PR TITLE
add customXml in the order list for rebuild

### PR DESCRIPTION
**Pull request by jdugh**
In addition to 'worksheet', 'theme', 'styles', there can be have `customXML`
If it is not in the order, there is a bug on the match relationShipId and the sheet Id in the workbook.
When I have more time, I while make a unitTest for this and maybe a better solution.

### CODE REVIEW

**Comment by kant2002** [comment]
Can you add test which verify this behaviour. I suspect you should just add file with that `customXML` content and make sure that it opens/processed.

**Comment by kant2002** [comment]
@jdugh friendly ping

**Comment by jdugh** [comment]
@kant2002 yes, I will make a test for this function, and also for the copy/delete sheet who haven't test.
I try to do this over the weekend

**Comment by kant2002** [comment]
@jdugh friendly ping.

File: lib/index.js [hunk 1]
```diff
@@ -148,6 +148,12 @@ module.exports = (function() {
       .sort(function(rel1, rel2){ //using order
         var index1 = order.indexOf( path.basename(rel1.attrib.Type) );
         var index2 = order.indexOf( path.basename(rel2.attrib.Type) );
+        // If the attrib.Type is not in the order list, go to the end of sort
+        // Maybe we can do it more gracefully with the boolean operator
+        if( index1 < 0 && index2 >= 0) return 1; // rel1 go after rel2
```
Comment (kant2002) [inline lib/index.js:153]: Nit: can you format braces to be consistent with file. 
```suggestion
        if (index1 < 0 && index2 >= 0) return 1; // rel1 go after rel2
```

**Review by kant2002 — COMMENTED** [review]
👍overall. Small nits

**Comment by jdugh** [comment]
Hi @kant2002 ,
That's it, I finally found the time to do the test !
Finally, with the test I found a more graceful solution for sort the relationShips: if it is not in the 'order' list, I push them to the end.
What do you think about that ?
Otherwise we would have to list all existing relationShip ?

**Comment by kant2002** [comment]
I think your solution is fine.

**Comment by kant2002** [comment]
Thank you!

**Pull request merged**